lib/Router.js

```diff
@@ -425,26 +427,11 @@ Router.prototype.handle = function(request, connection) {
 
     return Promise.all(callbacks)
     .then(function(results) {
-        results = results.map(function(result) {
-            // Multiple resources returned
-            if (_.isArray(result)) {
-                return result.map(function(r) {
-                    // Convert to an object if an instance
-                    return (_.isObject(r)) ? _.merge({}, r) : r;
-                });
-            } else if (_.isObject(result)) {
-                return _.merge({}, result);
-            }
-
-            return result;
-        });
+        debug('got results from callbacks', results);
```

Comment:
These debug statements are intended to be left in. This is using the `debug` module, which defaults to a noop unless you start the node process with `DEBUG=monocle-api:*`

See https://github.com/visionmedia/debug